CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,50 @@
+			Gayatri Vidya Parishad -Artificial Intelligence CLUB
+
+Community Code of Conduct
+
+The community and its members must, at all times, comply with all applicable rules and regulations. The community will not condone the activities of members who achieve results through violation of the rule or unethical practices.
+
+ Expected Behaviour:
+The community expects its members to conduct themselves in a professional manner. 
+
+The community expects that members will perform their duties conscientiously, honestly, and in accordance with the best interests of the community. 
+
+All members share a serious responsibility for the community’s good public relations, especially at the community level. Their readiness to help with religious, charitable, educational, and civic activities brings credit to the community and is encouraged. 
+
+Members who have access to community funds in any form must follow the prescribed procedures for recording, handling, and protecting money as detailed in the community’s policies and procedures or other explanatory materials, or both. 
```

Comment:
Good thing brought up! We don't have a policy for this yet.

Reply:
Yeah, should work on that